ehr/api-src/org/labkey/api/ehr/dataentry/forms/HousingFormType.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2014-2018 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.labkey.api.ehr.dataentry.forms;
+
+import org.json.old.JSONObject;
+import org.labkey.api.ehr.dataentry.AnimalDetailsFormSection;
+import org.labkey.api.ehr.dataentry.DataEntryFormContext;
+import org.labkey.api.ehr.dataentry.TaskFormSection;
+import org.labkey.api.ehr.dataentry.UnsaveableTask;
+import org.labkey.api.ehr.security.EHRHousingTransferPermission;
+import org.labkey.api.module.Module;
+import org.labkey.api.view.template.ClientDependency;
+
+import java.util.Arrays;
+
+/**
+ * User: bimber
+ * Date: 7/29/13
+ * Time: 5:03 PM
+ */
+public class HousingFormType extends UnsaveableTask
```

Comment:
Why are you adding this to core module if nobody is using it? Could we make it easier to leverage by letting each center plug in its down AnimalDetails section instead of having to create a new class?

Reply:
I removed it. I was originally going to use it but ended up not. Thought it would be ok to leave for base EHR but it is not being used so I'll remove it for now and think about a refactor.